client/src/components/App/AppToolbar/TopToolbar/TopToolbarStyles.ts

```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from '@material-ui/styles'
+
+const useStyles = makeStyles({
+    toolbar: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+
+    },
+    RightDiv: {
+        display: 'flex',
+        flexDirection: 'row',
+    },
+    logo: {
+        height: '8vh',
+    },
+    systemName: {
+        margin: '0.5vw',
+        fontSize: '2vw'
```

Comment:
Is any of the typography's built in variants fits here insted of using fontSize?